src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2315,6 +2377,97 @@
             "required": [ "name" ]
         },
 
+        "toolComponentTranslation": {
+            "description": "TBD.",
+            "additionalProperties": false,
+            "type": "object",
+            "properties": {
+
+                "toolComponentGuid": {
+                    "description": "TBD must be GUID format: matches toolComponent.guid.",
```

Comment:
>TBD [](start = 36, length = 3)

"The unique identifier for the tool component in the form of a GUID, matching toolComponent.guid."